inheritance/lib/src/test/java/inheritance/RestaurantReviews/RestaurantTest.java

```diff
@@ -9,8 +9,24 @@
 class RestaurantTest {
     @Test void testRestaurantCreation() {
 
-        Restaurant goodBites = new Restaurant("Good Bites", 4, 2);
-        assertEquals("Restaurant{name='Good Bites', stars=4.0, priceCategory=2, reviews=[]}", goodBites.toString());
```

Comment:
Make sure to use these assertions (`assertEquals`, `assertTrue`, etc.). Those are what actually test your code and tell you whether it passes or fails.